examples/src/main/java/ai/djl/examples/inference/Yolov8Detection.java

```diff
@@ -65,6 +65,8 @@ public static DetectedObjects predict() throws IOException, ModelException, Tran
         arguments.put("toTensor", true);
         arguments.put("applyRatio", true);
         arguments.put("threshold", 0.6f);
+        // for performance optimization maxBox parameter can reduce number of considered boxes from 8400
```

Comment:
We follow AOPS coding convention. Please reformat your code with the following command:

```
./gradlew fJ
```